src/Opdex.Platform.WebApi/Models/Responses/Transactions/TransactionEvents/TransactionEvent.cs

```diff
@@ -0,0 +1,69 @@
+using Newtonsoft.Json;
+using NJsonSchema.Converters;
+using Opdex.Platform.Common.Enums;
+using Opdex.Platform.Common.Models;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.Deployers;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.Governances;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.LiquidityPools;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.Markets;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.MiningPools;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.Tokens;
+using Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents.Vault;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Serialization;
+
+namespace Opdex.Platform.WebApi.Models.Responses.Transactions.TransactionEvents
+{
+    /// <summary>
+    /// Event that occurred in a transaction.
+    /// </summary>
+    [JsonConverter(typeof(JsonInheritanceConverter), "eventType")]
+    [KnownType(typeof(CreateMarketEvent))]
+    [KnownType(typeof(NominationEvent))]
```

Comment:
My ignorance might be showing here, but how come we can't keep the same ...ResponseModel names and include them here if this is mostly what the PR is doing?

Reply:
Unfortunately NSwag uses the type name to generate the spec and there's no way to customise this.